baybe/recommenders/pure/bayesian/base.py

```diff
@@ -68,7 +68,10 @@ def _get_acquisition_function_cls(
         return fun
 
     def setup_acquisition_function(
-        self, searchspace: SearchSpace, train_x: pd.DataFrame, train_y: pd.DataFrame
+        self,
+        searchspace: SearchSpace,
+        train_x: Optional[pd.DataFrame] = None,
```

Comment:
Why did you already make them optional if this is not supported yet?

Reply:
I didnt enable them (theres a NotImplemented error raised if they are none) but I had to change the signature for making mypy realize